common/execution_context.go

```diff
@@ -64,21 +80,31 @@ func NewExecutionContext(
 
 // Adopts specified backend node into this execution context from another execution context
 func (e *ExecutionContext) adoptBackendNodeId(backendNodeID cdp.BackendNodeID) (*ElementHandle, error) {
-	var remoteObj *runtime.RemoteObject
-	var err error
+	e.logger.Debugf("ExecutionContext:adoptBackendNodeId", "sid:%s tid:%s tid:%s ecid:%d furl:%q bnid:%d",
+		e.session.id, e.session.targetID, e.frame.id, e.id, e.frame.url, backendNodeID)
+
+	var (
+		remoteObj *runtime.RemoteObject
+		err       error
+	)
 
 	action := dom.ResolveNode().
 		WithBackendNodeID(backendNodeID).
 		WithExecutionContextID(e.id)
+
 	if remoteObj, err = action.Do(cdp.WithExecutor(e.ctx, e.session)); err != nil {
-		return nil, fmt.Errorf("unable to get DOM node: %w", err)
+		return nil, fmt.Errorf("cannot resolve dom node: %w", err)
```

Comment:
We should keep the original capitalization.
```suggestion
		return nil, fmt.Errorf("cannot resolve DOM node: %w", err)
```